tests/test-customize-changesets-controller.php

```diff
@@ -1384,55 +1299,41 @@ public function test_update_item_edit_not_future_date_with_past_date() {
 		) );
 		$response = $this->server->dispatch( $request );
 
-		$this->assertErrorResponse( 'not_future_date', $response );
+		$this->assertErrorResponse( 'rest_invalid_param', $response );
 		$this->assertSame( $date_before, get_post( $manager->changeset_post_id() )->post_date_gmt );
 	}
 
 	/**
 	 * Test that update_item() rejects scheduling a changeset when it has a past date.
+	 *
+	 * @todo Verify that this is true -- not clear why should be be rejected.
```

Comment:
The idea I think is that a changeset should only be able to have a date that it is the same or greater than the current time. When a changeset is transitioned to publish, it should always get a date that is then set to the current datetime, since the changeset then becomes a snapshot revision of the settings that were applied at that date/time. In that sense it doesn't make sense to date a changeset to a time in the past. So usually a changeset should either have its date set to empty, in which case it always gets the current date, or if a date is supplied then it must be a future date.